src/main/resources/static/js/asynchronousPro.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * JSONPlaceholderのPostsのベースURL
+ * @type {string}
+ */
+const BASE_URL = 'https://jsonplaceholder.typicode.com/posts/';
+
+/**
+ * 1~maxまでのランダムな整数を1つ返す関数
+ * @param {number} max ランダムな整数の最大値を指定
+ * @returns {number}
+ */
+function createRandomNumber(max){
+    return Math.ceil(Math.random() * max);
+}
+
+/**
+ * other.htmlの該当id部分を引数の値で書き換える関数
+ * @param {string} userId ユーザID
+ * @param {string} id ID
+ * @param {string} title タイトル
+ * @param {string} body 本文
+ */
+function setDataToView(userId, id, title, body) {
+    document.getElementById('userId').innerHTML = userId;
+    document.getElementById('id').innerHTML = id;
+    document.getElementById('title').innerHTML = title;
+    document.getElementById('body').innerHTML = body;
```

Comment:
意図的にHTMLタグの挿入を行いたいとき以外は`textContent`を利用するのがベターですね
```suggestion
    document.getElementById('body').textContent = body;
```
https://qiita.com/RyBB/items/c87af2413c34f9367d00

Reply:
こちら、よく調べずにinnerHTMLしかないものだと思っていました! 今回はテキストのみなのでこちらのほうがよさそうですね!